docs/content/TextInput.mdx

```diff
@@ -47,6 +47,65 @@ TextInput is a form component to add default styling to the native text input.
 </>
 ```
 
+## Text Input with visuals and loading indicators
```

Comment:
While we're in this file, can we nest all these examples under an `Examples` heading?

I also think we could remove `Text Input` from these headings (`Text Input with visuals... -> With visuals...`)

```
## Examples

### Default

### With icons

...
```